Orange/statistics/util.py

```diff
@@ -281,13 +281,13 @@ def mean(x):
     return np.sum(x.data) / n_values
 
 
-def nanmean(x):
+def nanmean(x, axis=None):
     """ Equivalent of np.nanmean that supports sparse or dense matrices. """
     if not sp.issparse(x):
-        return np.nanmean(x)
+        return np.nanmean(x, axis=axis)
 
-    n_values = np.prod(x.shape) - np.sum(np.isnan(x.data))
-    return np.nansum(x.data) / n_values
+    x.data = np.nan_to_num(x.data)
```

Comment:
`nan_to_num` converts `np.nans` to zeros, which causes `mean` to also treat them as zeros. E.g. for the sparse array of `[np.nan, np.nan, 1]` this implementation returns 0.33 instead of 1. 

What's wrong with the previous implementation?